docs/excel/custom-functions-parameter-options.md

```diff
@@ -28,31 +27,83 @@ Whereas regular parameters are required, optional parameters are not. When a use
  * @returns {number} The sum of the numbers.
  */
 function add(first, second, third) {
-  if (third !== undefined) {
-    return first + second + third;
+   if (third === null) {
+    third=0;
   }
-  return first + second;
+  return first + second + third;
 }
 CustomFunctions.associate("ADD", add);
 ```
 
-When you define a function that contains one or more optional parameters, you should specify what happens when the optional parameters are undefined. In the following example, `zipCode` and `dayOfWeek` are both optional parameters for the `getWeatherReport` function. If the `zipCode` parameter is undefined, the default value is set to `98052`. If the `dayOfWeek` parameter is undefined, it is set to Wednesday.
+#### [TypeScript](#tab/typescript)
+
+```typescript
+/**
+ * Add two numbers
+ * @customfunction 
```

Comment:
Also make same change (as described in previous suggestion) within the description of the **JavaScript** function.

(i.e., -> "Calculates the sum of the specified numbers")

Reply:
fixed.